ephios/settings.py

```diff
@@ -347,7 +347,6 @@ def GET_USERCONTENT_QUOTA():
 
 # Guardian configuration
 ANONYMOUS_USER_NAME = None
-GUARDIAN_MONKEY_PATCH = False
```

Comment:
why was this here and why don't we need it anymore?

Reply:
They renamed it, I reread the docs and I think we can reintroduce it under the new name. 
It disables monkeypatching the default User model, which we don't use, and which didn't cause any issues.